maven-projects/info/src/main/java/org/apache/graphar/info/yaml/GraphYaml.java

```diff
@@ -54,15 +54,26 @@ public GraphYaml(GraphInfo graphInfo) {
         this.prefix = graphInfo.getPrefix();
         this.vertices =
                 graphInfo.getVertexInfos().stream()
-                        .map(vertexInfo -> vertexInfo.getLabel() + ".vertex.yaml")
+                        .map(vertexInfo -> vertexInfo.getType() + ".vertex.yaml")
                         .collect(Collectors.toList());
         this.edges =
                 graphInfo.getEdgeInfos().stream()
                         .map(edgeInfo -> edgeInfo.getConcat() + ".edge.yaml")
                         .collect(Collectors.toList());
-        this.version = graphInfo.getVersion();
     }
 
+    //    public GraphInfo toGraphInfo(GraphLoader graphLoader) throws IOException {
```

Comment:
Could you add some comment that explain why comment the code?

Reply:
removed